code/modules/modular_computers/networking/_network.dm

```diff
@@ -197,3 +197,18 @@
 			allowed_mainframes |= D
 	return allowed_mainframes
 
+/datum/computer_network/proc/get_devices_by_type(var/type, var/mob/user)
+	var/bypass_auth = !user
+	if(!bypass_auth)
```

Comment:
Not sure what all this bypass_auth stuff is for

Reply:
When a user is not provided all authentication is bypassed. Don't worry about it, the commit it's a part of is nebula mainstream cherrypick.